docs/getting-started.md

```diff
@@ -0,0 +1,100 @@
+# Getting Started
+
+Welcome to the Getting Started guide for the refarch-templates. This guide
+is divided into sections for Frontend/Web Component and Backend/EAI, as well
+as a general setup and requirements applicable to all templates.
+
+## General
+
+To use the refarch-templates, you need to have Docker installed on your
+machine. Inside the `stack` folder, you will find a `docker-compose.yml`
+file that will spin up everything needed for local development. 
+
+When developing locally, the stack should always be running. 
+### Download Templates
```

Comment:
I'd add documentation for the alternative way. Instead of cloning it, users could also use the template directly via GitHub, as we provided the repository as a "GitHub template repository"

Reply:
that kinda depends, is that the way people should use it? If so, can we chance the interel process, so that a new project that is based on the refarch will get automatically get the template? Should discuss this internally first.